toggles-sample/src/main/AndroidManifest.xml

```diff
@@ -3,6 +3,13 @@
     xmlns:tools="http://schemas.android.com/tools"
     package="com.example.wrench">
 
+<!--    <queries>-->
```

Comment:
just curious why this change if the code not gonna to be in use ? 😄

Reply:
I added it when trying to figure out why the heck everything stopped working.. :) was blissfully unaware of the changes made.. I'll make sure to remove it once I get around to take a next step with this PR (updated values notifications aren't propagating properly either - seems that's been changed too)

Reply:
I see, nice 👍

Reply:
gone!